### ISSUE TITLE
Failure to build current academic-kickstarter due to lack of hugo extended

### ISSUE THREAD

**Issue by sloede** [benmatselby/hugo-deploy-gh-pages#27]
When trying to deploy the current [academic-kickstart](https://github.com/sourcethemes/academic-kickstart) repo, the action fails with an error that *seems* to indicate that Hugo Extended is required:
```
Error: Error building site: TOCSS: failed to transform "main_parsed.scss" (text/x-scss): resource "scss/scss/main.scss_6c95cc1249b26b124274204dbf970c34" not found in file cache
[...]
```
According to [this issue](https://github.com/gcushen/hugo-academic/issues/1092#issuecomment-493787281), Hugo Extended *does* seem to be a prerequisite. However, in the current version of this action there does not seem to be a way to specify that Hugo Extended should be used.

**Comment by jrwrigh** [comment]
I've never had any issues with the academic-kickstart repo needing anything but vanilla hugo. 
But [the release notes from the most recent version of academic](https://sourcethemes.com/academic/updates/v4.7.0#breaking-changes) do suggest the extended version is necessary.

**Comment by benmatselby** [comment]
Resolved by #28. Thanks for the PR @sloede 👍
